nixos/modules/services/misc/gitlab.nix

```diff
@@ -10,9 +10,10 @@ let
   ruby = cfg.packages.gitlab.ruby;
   bundler = pkgs.bundler;
 
-  gemHome = "${cfg.packages.gitlab.env}/${ruby.gemPath}";
+  gemHome = "${cfg.packages.gitlab.ruby-env}/${ruby.gemPath}";
```

Comment:
There are two envs, a node one and a ruby one. It's easier to call it `ruby-env` to not mess up. Though with the suggestions from @fpletz, I might be able to remove the `node-env` altogether.